docs/en/observability/configure-logs-sources.asciidoc

```diff
@@ -23,7 +23,11 @@ default configuration settings.
 
 | *Name* | Name of the source configuration. 
 
-| *Indices* | Index pattern or patterns in the {es} indices to read log data from.
+| *Index pattern* | Index pattern or patterns in the {es} indices to read log data from.
+
+Each log source now integrates with {kib} index patterns which support creating and
```

Comment:
What are *Kibana index patterns*? This term was added to our "What's New" doc as well, but I can't find any mention of it in the [runtime fields docs](https://www.elastic.co/guide/en/kibana/7.13/managing-index-patterns.html), or anywhere else in the Kibana guide.